expect.go

```diff
@@ -94,3 +94,20 @@ func (t *T) ExpectErrorMessagef(err error, msg string, spec string, args ...inte
 			"Error message=%s\nExpected string=%s", prefix, err.Error(), msg)
 	}
 }
+
+// Expects the function passed in to panic. This will call f() and expect
+// that an error matching err will be raised as a panic.
+func (t *T) ExpectPanic(f func(), err interface{}, desc ...string) {
```

Comment:
This shouldn't be called `err`, since it's not `error` but "whatever was panic'd"

Reply:
Very true.. addressed here: https://github.com/liquidgecka/testlib/pull/7